net-test/bin/config.sh

```diff
@@ -0,0 +1,26 @@
+# local config
+__ROOT="$(realpath "$(pwd)"/..)"
+__ETC="$_ROOT/etc"
```

Comment:
there is an `_` missing on this line and the second line for using `__ROOT`

Reply:
Fixed